docs/opentracing.asciidoc

```diff
@@ -1,19 +1,19 @@
 [[opentracing]]
 == OpenTracing API
 
-The Elastic APM Go agent provides an implementation of the https://opentracing.io[OpenTracing API],
+The Elastic APM Go {agent} provides an implementation of the https://opentracing.io[OpenTracing API],
```

Comment:
I'm not sure you want to use the `{agent}` attribute. This would resolve to Elastic APM Go Elastic Agent. @brandon can confirm if this is the correct terminology.